installation failed on MacOS

please find below the steps I followed:

I installed the dependencies first, and then gem install, it failed, I tried `brew link gettext --force`, still same error.

```
  650  brew install glib gdk-pixbuf cairo pango cmake
  651  cd ~/Library/Fonts
  652  ll
  653  curl -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/cmex10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/cmmi10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/cmr10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/cmsy10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/esint10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/eufm10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/msam10.ttf      -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/msbm10.ttf
  654  gem install asciidoctor-mathematical
  655  brew link gettext --force
  656  brew unlink gettext && brew link gettext
  657  gem install asciidoctor-mathematical

```	
	
Please find below the error message for gem install.

```
WangXus-MacBook-Pro:Fonts wangxu$ gem install asciidoctor-mathematical
Building native extensions. This could take a while...
ERROR:  Error installing asciidoctor-mathematical:
	ERROR: Failed to build gem native extension.

    current directory: /Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical
/Users/wangxu/.rubies/ruby-2.7.0/bin/ruby -I /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0 -r ./siteconf20200504-47590-1cgil0b.rb extconf.rb
checking for cmake... yes
checking for -lxml2... yes
checking for -lpangocairo-1.0... yes
checking for libxml/tree.h in /include/libxml2,/usr/include/libxml2,/usr/local/include/libxml2,/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libxml2... yes
checking for libxml/parser.h in /include/libxml2,/usr/include/libxml2,/usr/local/include/libxml2,/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libxml2... yes
checking for libxml/xpath.h in /include/libxml2,/usr/include/libxml2,/usr/local/include/libxml2,/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libxml2... yes
checking for libxml/xpathInternals.h in /include/libxml2,/usr/include/libxml2,/usr/local/include/libxml2,/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libxml2... yes
-- The C compiler identification is AppleClang 11.0.0.11000033
-- The CXX compiler identification is AppleClang 11.0.0.11000033
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found BISON: /usr/bin/bison (found version "2.3")
-- Found FLEX: /usr/bin/flex (found version "2.5.35")
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Found PythonInterp: /Users/wangxu/.pyenv/shims/python (found version "3.7.7")
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/mtex2MML/build
[ 10%] [FLEX][MTEX2MMLSCANNER] Building scanner with flex 2.5.35
[ 20%] [BISON][MTEX2MMLPARSER] Building parser with bison 2.3
conflicts: 5280 shift/reduce, 613 reduce/reduce
Scanning dependencies of target libmtex2MML_static
[ 30%] Building C object CMakeFiles/libmtex2MML_static.dir/src/colors.c.o
[ 40%] Building C object CMakeFiles/libmtex2MML_static.dir/src/em.c.o
[ 50%] Building C object CMakeFiles/libmtex2MML_static.dir/src/environment.c.o
[ 60%] Building C object CMakeFiles/libmtex2MML_static.dir/src/string_dup.c.o
[ 70%] Building C object CMakeFiles/libmtex2MML_static.dir/src/string_extras.c.o
[ 80%] Building C object CMakeFiles/libmtex2MML_static.dir/parser.c.o
/Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/mtex2MML/src/parser.y:56:30: warning: unused parameter 'ret_str' [-Wunused-parameter]
 static void yyerror (char **ret_str, const char * s)
                             ^
/Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/mtex2MML/src/parser.y:68:14: warning: unused function 'mtex2MML_default_write' [-Wunused-function]
 static void mtex2MML_default_write (const char * buffer, unsigned long length)
             ^
/Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/mtex2MML/src/parser.y:79:14: warning: unused function 'mtex2MML_default_write_mathml' [-Wunused-function]
 static void mtex2MML_default_write_mathml (const char * mathml)
             ^
3 warnings generated.
[ 90%] Building C object CMakeFiles/libmtex2MML_static.dir/lexer.c.o
/Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/mtex2MML/build/lexer.c:8203:16: warning: function 'input' is not needed and will not be emitted [-Wunneeded-internal-declaration]
    static int input  (void)
               ^
1 warning generated.
[100%] Linking C static library libmtex2MML.a
[100%] Built target libmtex2MML_static
-- The C compiler identification is AppleClang 11.0.0.11000033
-- The CXX compiler identification is AppleClang 11.0.0.11000033
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2")
-- Checking for module 'glib-2.0'
--   Found glib-2.0, version 2.64.2
-- Checking for module 'cairo'
--   Found cairo, version 1.12.2
-- Checking for module 'pango'
--   Package 'pango' requires 'fontconfig >= 2.11.91' but version of fontconfig is 2.10.1
CMake Error at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/FindPkgConfig.cmake:467 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/FindPkgConfig.cmake:647 (_pkg_check_modules_internal)
  CMakeLists.txt:43 (pkg_check_modules)


-- Configuring incomplete, errors occurred!
See also "/Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/lasem/build/CMakeFiles/CMakeOutput.log".
make: *** No targets specified and no makefile found.  Stop.
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
	--with-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/Users/wangxu/.rubies/ruby-2.7.0/bin/$(RUBY_BASE_NAME)
	--with-xml2-dir
	--without-xml2-dir
	--with-xml2-include
	--without-xml2-include=${xml2-dir}/include
	--with-xml2-lib
	--without-xml2-lib=${xml2-dir}/lib
	--with-xml2lib
	--without-xml2lib
	--with-pangocairo-1.0-dir
	--without-pangocairo-1.0-dir
	--with-pangocairo-1.0-include
	--without-pangocairo-1.0-include=${pangocairo-1.0-dir}/include
	--with-pangocairo-1.0-lib
	--without-pangocairo-1.0-lib=${pangocairo-1.0-dir}/lib
	--with-pangocairo-1.0lib
	--without-pangocairo-1.0lib
	--use-system-mtex2MML
	--use-system-lasem
/Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:491:in `realpath': No such file or directory @ rb_check_realpath_internal - /Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13/ext/mathematical/lasem/build/liblasem.dylib (Errno::ENOENT)
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:491:in `copy_entry'
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:468:in `block in cp_r'
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:1588:in `block in fu_each_src_dest'
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:1602:in `fu_each_src_dest0'
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:1586:in `fu_each_src_dest'
	from /Users/wangxu/.rubies/ruby-2.7.0/lib/ruby/2.7.0/fileutils.rb:467:in `cp_r'
	from extconf.rb:99:in `<main>'

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /Users/wangxu/.gem/ruby/2.7.0/extensions/x86_64-darwin-19/2.7.0-static/mathematical-1.6.13/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /Users/wangxu/.gem/ruby/2.7.0/gems/mathematical-1.6.13 for inspection.
Results logged to /Users/wangxu/.gem/ruby/2.7.0/extensions/x86_64-darwin-19/2.7.0-static/mathematical-1.6.13/gem_make.out
WangXus-MacBook-Pro:Fonts wangxu$
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,9 +17,9 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         ruby: ['2.3', '2.4', '2.5', '2.6', '2.7']
-        #include:
-        #- os: macos-latest
-        #  ruby: '2.7'
+        include:
+        - os: macos-latest
+          ruby: '2.7'
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
@@ -33,9 +33,9 @@ jobs:
     - name: Install system dependencies (Linux)
       if: matrix.os == 'ubuntu-latest'
       run: sudo apt-get install bison flex libffi-dev libxml2-dev libgdk-pixbuf2.0-dev libcairo2-dev libpango1.0-dev fonts-lyx
-    #- name: Install system dependencies (macOS)
-    #  if: matrix.os == 'macos-latest'
-    #  run: brew install cairo gdk-pixbuf pango
+    - name: Install system dependencies (macOS)
+      if: matrix.os == 'macos-latest'
+      run: brew install cairo gdk-pixbuf pango
     - name: Install dependencies
       run: bundle --jobs 3 --retry 3
     - name: Convert sample document

